### PR TITLE
PR: Update footer config

### PR DESCRIPTION
**Pull request by dalthviz**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [ ] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

A preview:

image

Depends on https://github.com/spyder-ide/spyder-docs-sphinx-theme/pull/9

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -139,8 +139,13 @@
...
     ],
     "use_edit_page_button": True,
+    "show_powered_by": True,
     "gitter_room": "spyder-ide/public",
-    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version"
+    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version",
+    "github_url": "https://github.com/pandas-dev/pandas",
+    "twitter_url": "https://twitter.com/pandas_dev",
+    "facebook_url": "https://www.facebook.com/SpyderIDE/",
+    "google_groups_url": "https://groups.google.com/group/spyderlib"
```
Comment (CAM-Gerlach) [inline doc/conf.py:148]:
```suggestion
    "google_groups_url": "https://groups.google.com/group/spyderlib",
```

Cleaner diffs and avoid potential bugs

File: doc/conf.py [hunk 2]
```diff
@@ -139,8 +139,28 @@
...
     "use_edit_page_button": True,
+    "show_powered_by": True,
     "gitter_room": "spyder-ide/public",
-    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version"
+    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version",
+    "open_collective": "spyder",
+    "footer_links": [
+        {
+            "url": "https://github.com/pandas-dev/pandas",
+            "name": "GitHub"
```
Comment (CAM-Gerlach) [inline doc/conf.py:149]:
```suggestion
            "name": "GitHub",
```

File: doc/conf.py [hunk 3]
```diff
@@ -139,8 +139,28 @@
...
+        },
+        {
+            "url": "https://twitter.com/pandas_dev",
+            "name": "Twitter"
```
Comment (CAM-Gerlach) [inline doc/conf.py:153]:
```suggestion
            "name": "Twitter",
```

File: doc/conf.py [hunk 4]
```diff
@@ -139,8 +139,28 @@
...
+        },
+        {
+            "url": "https://www.facebook.com/SpyderIDE/",
+            "name": "Facebook"
```
Comment (CAM-Gerlach) [inline doc/conf.py:157]:
```suggestion
            "name": "Facebook",
```

File: doc/conf.py [hunk 5]
```diff
@@ -139,8 +139,28 @@
...
+        },
+        {
+            "url": "https://groups.google.com/group/spyderlib",
+            "name": "Google Groups"
```
Comment (CAM-Gerlach) [inline doc/conf.py:161]:
```suggestion
            "name": "Google Groups",
```

File: doc/conf.py [hunk 6]
```diff
@@ -139,8 +139,28 @@
...
+        },
+        {
+            "url": "https://twitter.com/pandas_dev",
+            "name": "Twitter"
+        },
+        {
+            "url": "https://www.facebook.com/SpyderIDE/",
+            "name": "Facebook"
+        },
+        {
```
Comment (CAM-Gerlach) [inline doc/conf.py:159]:
```suggestion
        {
            "url": "https://instagram.com/spyderide",
            "name": "Instagram",
        },
        {
```

Missing our instagram that the main site has

**Comment by dalthviz** [comment]
Yep @CAM-Gerlach (by doing a re-run of the checks after the merge of https://github.com/spyder-ide/spyder-docs-sphinx-theme/pull/9 to pip install the update version of theme repo branch).

File: doc/conf.py [hunk 7]
```diff
@@ -139,8 +139,36 @@
...
     ],
     "use_edit_page_button": True,
+    "show_powered_by": True,
     "gitter_room": "spyder-ide/public",
-    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version"
+    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version",
+    "open_collective": "spyder",
+    "footer_links": [
+        {
+            "url": "https://github.com/pandas-dev/pandas",
```
Comment (ccordoba12) [inline doc/conf.py:148]:
```suggestion
            "url": "https://github.com/spyder-ide/spyder",
```

File: doc/conf.py [hunk 8]
```diff
@@ -139,8 +139,36 @@
...
+            "name": "GitHub",
+        },
+        {
+            "url": "https://twitter.com/pandas_dev",
```
Comment (ccordoba12) [inline doc/conf.py:152]:
```suggestion
            "url": "https://twitter.com/Spyder_IDE",
```

**Comment by CAM-Gerlach** [comment]
> Links to Github and Twitter were wrong.

That's a very embarrassing one for me to miss :facepalm: Thanks for the catch @ccordoba12

**Comment by ccordoba12** [comment]
No prob, that's why it's always better to have another pair of eyes reviewing PRs (@dalthviz also missed that one).

**Pull request merged**